test/scraped/response/decorator/absolute_urls_test.rb

```diff
@@ -134,4 +148,8 @@ def link(selector)
   it 'should not encode already encoded URLs' do
     page.encoded_url.must_equal 'http://example.com/person%20123'
   end
+
+  it 'should use a base url provided by the document' do
+    page_with_base.relative_link.must_equal 'http://www.example.com/person-123'
+  end
```

Comment:
Rather than just interspersing a whole new test setup and assertion throughout the existing test, you should scope each of these, so that the ones using the same page are grouped together, but separate from those using a different page.